src/client/sandbox/code-instrumentation/location/wrapper.js

```diff
@@ -135,6 +137,40 @@ export default class LocationWrapper extends EventEmitter {
             }
         }));
 
+        if (window.location.ancestorOrigins) {
+            this.messageIdGenerator  = createIntegerIdGenerator();
+            this.GET_ORIGIN_CMD      = 'hammerhead|command|get-origin';
```

Comment:
Move the `GET_ORIGIN_CMD` and `ORIGIN_RECEIVED_CMD` after the imports block as const